src/cordova/setup.action.mjs

```diff
@@ -63,6 +63,12 @@ export async function main(...parameters) {
     );
   }
 
+  if (platform === 'android') {
+    console.warn(
```

Comment:
This is noise if the user already has android setup. We should error if android is not setup and the error is not clear, but be silent if everything is in order.